go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/loft-sh/vcluster
 
-go 1.23.2
+go 1.24.0
```

Comment:
1.24.1 has been also released on 4th of March, but it's still not widely available in different package managers, so I didn't use it here ATM.